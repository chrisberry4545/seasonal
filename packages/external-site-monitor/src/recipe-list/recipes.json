[{"linkUrl":"https://www.momskitchenhandbook.com/uncategorized/pasta-with-creamy-beet-sauce-walnuts-and-parmesan/"},{"linkUrl":"https://www.thespruceeats.com/kale-and-cabbage-slaw-with-dijon-mustard-vinaigrette-4123786"},{"linkUrl":"http://allrecipes.co.uk/recipe/24491/carrot-and-parsnip-soup.aspx"},{"linkUrl":"http://goboldwithbutter.com/brussels-sprouts-and-mushroom-casserole/"},{"linkUrl":"http://cookwithmands.com/recipes/parsnip-and-leek-gratin"},{"linkUrl":"https://valentinascorner.com/cabbage-cucumber-radish-salad/"},{"linkUrl":"https://www.bbc.com/food/recipes/rhubarbcrumble_11396"},{"linkUrl":"http://eatsimplyeatwell.com/2014/10/celeriac-cabbage-carrot-coleslaw/"},{"linkUrl":"https://www.bbcgoodfood.com/recipes/new-potatoes-spinach-capers"},{"linkUrl":"https://www.bbcgoodfood.com/recipes/1267639/peppered-mackerel-new-potato-and-watercress-salad"},{"linkUrl":"http://www.forkknifeswoon.com/weeknight-kale-spinach-goat-cheese-pasta/"},{"linkUrl":"https://msmarket.coop/recipe/spinach-kale-soup/"},{"linkUrl":"https://www.recipetineats.com/rocket-aragula-beetroot-walnuts-feta-wbalsamic-dressing/"},{"linkUrl":"https://www.bbcgoodfood.com/recipes/eton-mess-cheesecake"},{"linkUrl":"https://healthylivingjames.co.uk/10-min-chickpea-spinach-curry/"},{"linkUrl":"https://www.thekitchn.com/how-to-make-stuffed-peppers-234387"},{"linkUrl":"https://www.bbc.com/food/recipes/how_to_make_pea_and_ham_17426"},{"linkUrl":"https://www.theocooks.com/spicy-aubergine-stir-fry/"},{"linkUrl":"https://www.mommypotamus.com/elderberry-jam-recipe/"},{"linkUrl":"https://www.foodiequine.co.uk/2015/07/strawberry-blackcurrant-coconut-water.html"},{"linkUrl":"https://www.pillsbury.com/recipes/cucumber-pepper-salad/1159ed6b-7863-4b60-b4e1-ce95520c80dc"},{"linkUrl":"https://pickledplum.com/smacked-cucumber-chili-oil/"},{"linkUrl":"https://www.foodandwine.com/recipes/broccoli-frittata"},{"linkUrl":"https://www.bbcgoodfood.com/recipes/2493639/spanish-stuffed-marrow"},{"linkUrl":"https://www.bbcgoodfood.com/recipes/1396633/courgette-broccoli-and-gremolata-pasta"},{"linkUrl":"https://www.bbcgoodfood.com/recipes/burrata-bruschetta-summer-beans"},{"linkUrl":"http://allrecipes.co.uk/recipe/1551/garlic-green-beans.aspx"},{"linkUrl":"https://www.chilipeppermadness.com/chili-pepper-recipes/breakfasts/shakshuka-eggs-poached-in-spicy-tomato-sauce/"},{"linkUrl":"https://www.bbc.com/food/recipes/chilliconcarne_67875"},{"linkUrl":"https://www.dinneratthezoo.com/ground-beef-tacos/"},{"linkUrl":"https://www.jamieoliver.com/recipes/vegetables-recipes/ratatouille/"},{"linkUrl":"https://www.bbcgoodfood.com/recipes/2358637/sausage-fennel-and-rocket-fusilli"},{"linkUrl":"https://realfood.tesco.com/recipes/potato-and-corn-chowder.html"},{"linkUrl":"https://reciperunner.com/roasted-butternut-squash-pumpkin-soup/"},{"linkUrl":"https://www.bbcgoodfood.com/recipes/italian-sausage-chestnut-pasta"},{"linkUrl":"https://www.bbcgoodfood.com/recipes/358608/cauliflower-cheese"},{"linkUrl":"https://www.bbcgoodfood.com/recipes/asparagus-chilli-feta-farfalle"},{"linkUrl":"https://www.bbcgoodfood.com/recipes/butter-poached-asparagus-leeks-peas"},{"linkUrl":"https://www.jamieoliver.com/recipes/potato-recipes/easy-rustic-gnocchi/"},{"linkUrl":"https://www.bbcgoodfood.com/recipes/pea-broad-bean-shakshuka"},{"linkUrl":"https://www.bbcgoodfood.com/recipes/broad-bean-feta-watercress-salad"},{"linkUrl":"https://www.foodandwine.com/recipes/spicy-carrot-sandwiches"},{"linkUrl":"http://www.foodnetwork.co.uk/recipes/honey-glazed-carrots.html?utm_source=foodnetwork.com&utm_medium=domestic"},{"linkUrl":"https://www.connoisseurusveg.com/gingery-carrot-stew-with-peanuts-and-cilantro/"},{"linkUrl":"https://www.kitchenhealssoul.com/red-currant-muffins/"},{"linkUrl":"https://realfood.tesco.com/recipes/carrot-and-chickpea-stew.html"},{"linkUrl":"https://www.tasteofhome.com/recipes/warm-rice-pintos-salad/"},{"linkUrl":"https://www.sbs.com.au/food/recipes/spicy-korean-noodles-cucumber-and-egg-bibim-guksu"},{"linkUrl":"https://www.bbcgoodfood.com/recipes/ultimate-tomato-salsa"},{"linkUrl":"https://thehappyfoodie.co.uk/recipes/courgette-and-aubergine-curry"},{"linkUrl":"https://www.bbcgoodfood.com/recipes/10603/roasted-vegetable-lasagne"},{"linkUrl":"https://neighborfoodblog.com/2014/07/spicy-asian-zucchini-side-dish.html"},{"linkUrl":"http://resourcefulcook.com/recipes/view/ylo26-pepper-courgette-stir-fry"},{"linkUrl":"https://www.healthyseasonalrecipes.com/simple-kohlrabi-with-pistachios-and-sage/"},{"linkUrl":"https://lasbobba.com/2015/04/23/carrot-radish-and-hummus-sandwich/"},{"linkUrl":"https://www.riverford.co.uk/recipes/view/recipe/runner-bean-ratatouille"},{"linkUrl":"https://www.bbcgoodfood.com/recipes/gnocchi-parsley-butter-samphire"},{"linkUrl":"https://www.bbcgoodfood.com/recipes/2220641/squid-samphire-and-pea-salad"},{"linkUrl":"https://www.bonappetit.com/recipe/turnip-and-kale-gratin"},{"linkUrl":"https://eatfresh.org/recipe/side-dish-snacks-breakfast/turnip-pancakes-latkes#.XSC6vNP0nBI"},{"linkUrl":"https://itsavegworldafterall.com/roasted-turnip-hummus/"},{"linkUrl":"https://www.bosh.tv/recipes/quick-and-easy-moroccan-couscous"},{"linkUrl":"https://www.channel4.com/programmes/sunday-brunch/articles/all/cauliflower-katsu/6407"},{"linkUrl":"https://www.healthyseasonalrecipes.com/simple-sauteed-swiss-chard/"},{"linkUrl":"https://www.bonappetit.com/recipe/grilled-artichokes-with-yogurt-dill-dipping-sauce"},{"linkUrl":"https://recipes.timesofindia.com/recipes/orange-sauce/rs58773589.cms"},{"linkUrl":"https://www.bbcgoodfood.com/recipes/9088/bestever-chunky-guacamole"},{"linkUrl":"https://ottolenghi.co.uk/french-beans-with-shiitake-mushrooms-nutmeg-shop"},{"linkUrl":"https://www.taste.com.au/recipes/fig-ginger-gratin/inu15was"},{"linkUrl":"https://iheartvegetables.com/honeydew-sorbet/"},{"linkUrl":"https://littlesunnykitchen.com/loquat-vodka-cocktail/"},{"linkUrl":"https://www.delicious.com.au/recipes/mango-avocado-black-lentil-salad/HsSi2VPv?r=recipes/collections/mangorecipes"},{"linkUrl":"http://www.sweetpaulmag.com/food/nectarine-spritzer"},{"linkUrl":"https://www.bbcgoodfood.com/recipes/passionfruit-pornstar-martini"},{"linkUrl":"https://www.sbs.com.au/food/recipes/swede-and-silverbeet-pie"},{"linkUrl":"https://www.chinasichuanfood.com/chinese-style-cabbage-stir-fry/"},{"linkUrl":"https://www.taste.com.au/recipes/ratatouille-5/44db8695-c005-4db1-b874-ba3ffa7c3116"},{"linkUrl":"https://www.donnahay.com.au/recipes/banana-bread"},{"linkUrl":"https://www.sbs.com.au/food/recipes/blackcurrant-jam"},{"linkUrl":"https://www.bbcgoodfood.com/recipe/blueberry-pretzel-cookies"},{"linkUrl":"https://cookieandkate.com/naturally-sweetened-cranberry-sauce-recipe/"},{"linkUrl":"https://sallysbakingaddiction.com/grapefruit-greek-yogurt-loaf/"},{"linkUrl":"https://www.jamieoliver.com/recipes/potato-recipes/easy-rustic-gnocchi/"},{"linkUrl":"https://www.bbcgoodfood.com/recipes/tomato-burrata-broad-bean-salad"},{"linkUrl":"https://www.olivemagazine.com/recipes/family/cherry-jam-and-almond-galettes/"},{"linkUrl":"https://www.waitrose.com/home/recipes/recipe_directory/s/scorched-leeks-withtomatoandcaperdressing.html"},{"linkUrl":"https://www.foodnetwork.ca/recipe/grilled-peach-panzanella-salad/18397/"},{"linkUrl":"https://realfood.tesco.com/recipes/vegan-nut-roast.html"},{"linkUrl":"https://www.mydiaryofus.com/blog/2015/5/14/plum-and-thyme-prosecco-smash"},{"linkUrl":"https://www.bosh.tv/recipes/butternut-squash-dahl-veganuary-week-2"},{"linkUrl":"https://deliciouslyella.com/2017/08/11/creamy-garlic-thyme-mushrooms/"},{"linkUrl":"https://momsdish.com/recipe/667/broccoli-mushroom-alfredo-pasta"},{"linkUrl":"https://www.thespruceeats.com/sauteed-radicchio-recipe-2217562"},{"linkUrl":"https://www.myrecipes.com/recipe/escarole-with-bacon-white-beans"},{"linkUrl":"http://www.eatingwell.com/recipe/259685/broccoli-with-balsamic-mushrooms/"},{"linkUrl":"https://www.delish.com/holiday-recipes/thanksgiving/a23104970/apple-pie-bars-recipe/"},{"linkUrl":"https://natashaskitchen.com/horseradish-sauce-recipe/"},{"linkUrl":"https://www.deliciousmagazine.co.uk/recipes/roast-potatoes-with-sage/"},{"linkUrl":"https://jessthomson.wordpress.com/2011/04/26/stung/"},{"linkUrl":"https://www.alaskafromscratch.com/2013/04/02/garlic-sesame-snow-peas/"},{"linkUrl":"https://www.olivemagazine.com/recipes/quick-and-easy/caramel-poached-apricots/"},{"linkUrl":"https://www.thespruceeats.com/asian-pear-crisp-2216135"},{"linkUrl":"https://www.bbcgoodfood.com/recipes/frozen-blackberry-yogurt"},{"linkUrl":"https://www.marthastewart.com/1047287/buckwheat-waffles-fresh-ricotta-and-citrus"},{"linkUrl":"https://savoryspin.com/spicy-breadfruit-fries/"},{"linkUrl":"https://www.finecooking.com/recipe/broccoflower-and-cremini-mushrooms-with-garlic-and-rosemary"},{"linkUrl":"https://www.myrecipes.com/recipe/green-beans-with-goat-cheese-tomatoes-almonds"},{"linkUrl":"https://www.tastymediterraneo.com/tagliatelle-with-fresh-oregano-pesto/"},{"linkUrl":"https://www.tasteofhome.com/recipes/tomato-garlic-butter-bean-dinner/"},{"linkUrl":"https://nadialim.com/recipe/whole-roasted-buttercup-squash/"},{"linkUrl":"https://www.greatbritishchefs.com/recipes/chicory-recipe"},{"linkUrl":"http://appetiteforchina.com/recipes/recipes/chinese-stir-fried-water-spinach"},{"linkUrl":"https://foodandstyle.com/green-bean-and-fresh-chickpea-salad-with-heirloom-tomatoes-and-fresh-mint/"}]