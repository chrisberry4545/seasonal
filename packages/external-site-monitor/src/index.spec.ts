import fetch from 'node-fetch';
import recipes from './recipe-list/recipes.json';

describe('check recipes sites have not changed', () => {

  const getStatus = async (
    url: string
  ): Promise<number> => {
    const site = await fetch(url);
    return site.status;
  };

  beforeAll(() => jest.setTimeout(50000));

  test.each(recipes)('%p recipe appears to still exist', async (recipe) => {
    const status = await getStatus(recipe.linkUrl);
    expect(status).toMatchSnapshot(200);
  });

});
