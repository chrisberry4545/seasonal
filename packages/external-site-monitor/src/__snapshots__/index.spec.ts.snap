// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`check recipes sites have not changed {"linkUrl": "http://allrecipes.co.uk/recipe/1551/garlic-green-beans.aspx"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "http://allrecipes.co.uk/recipe/24491/carrot-and-parsnip-soup.aspx"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "http://appetiteforchina.com/recipes/recipes/chinese-stir-fried-water-spinach"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "http://cookwithmands.com/recipes/parsnip-and-leek-gratin"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "http://eatsimplyeatwell.com/2014/10/celeriac-cabbage-carrot-coleslaw/"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "http://goboldwithbutter.com/brussels-sprouts-and-mushroom-casserole/"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "http://resourcefulcook.com/recipes/view/ylo26-pepper-courgette-stir-fry"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "http://www.eatingwell.com/recipe/259685/broccoli-with-balsamic-mushrooms/"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "http://www.foodnetwork.co.uk/recipes/honey-glazed-carrots.html?utm_source=foodnetwork.com&utm_medium=domestic"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "http://www.forkknifeswoon.com/weeknight-kale-spinach-goat-cheese-pasta/"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "http://www.sweetpaulmag.com/food/nectarine-spritzer"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://cookieandkate.com/naturally-sweetened-cranberry-sauce-recipe/"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://deliciouslyella.com/2017/08/11/creamy-garlic-thyme-mushrooms/"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://eatfresh.org/recipe/side-dish-snacks-breakfast/turnip-pancakes-latkes#.XSC6vNP0nBI"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://foodandstyle.com/green-bean-and-fresh-chickpea-salad-with-heirloom-tomatoes-and-fresh-mint/"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://healthylivingjames.co.uk/10-min-chickpea-spinach-curry/"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://iheartvegetables.com/honeydew-sorbet/"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://itsavegworldafterall.com/roasted-turnip-hummus/"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://jessthomson.wordpress.com/2011/04/26/stung/"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://lasbobba.com/2015/04/23/carrot-radish-and-hummus-sandwich/"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://littlesunnykitchen.com/loquat-vodka-cocktail/"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://momsdish.com/recipe/667/broccoli-mushroom-alfredo-pasta"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://msmarket.coop/recipe/spinach-kale-soup/"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://nadialim.com/recipe/whole-roasted-buttercup-squash/"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://natashaskitchen.com/horseradish-sauce-recipe/"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://neighborfoodblog.com/2014/07/spicy-asian-zucchini-side-dish.html"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://ottolenghi.co.uk/french-beans-with-shiitake-mushrooms-nutmeg-shop"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://pickledplum.com/smacked-cucumber-chili-oil/"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://realfood.tesco.com/recipes/carrot-and-chickpea-stew.html"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://realfood.tesco.com/recipes/potato-and-corn-chowder.html"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://realfood.tesco.com/recipes/vegan-nut-roast.html"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://reciperunner.com/roasted-butternut-squash-pumpkin-soup/"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://recipes.timesofindia.com/recipes/orange-sauce/rs58773589.cms"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://sallysbakingaddiction.com/grapefruit-greek-yogurt-loaf/"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://savoryspin.com/spicy-breadfruit-fries/"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://thehappyfoodie.co.uk/recipes/courgette-and-aubergine-curry"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://valentinascorner.com/cabbage-cucumber-radish-salad/"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.alaskafromscratch.com/2013/04/02/garlic-sesame-snow-peas/"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.bbc.com/food/recipes/chilliconcarne_67875"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.bbc.com/food/recipes/how_to_make_pea_and_ham_17426"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.bbc.com/food/recipes/rhubarbcrumble_11396"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.bbcgoodfood.com/recipe/blueberry-pretzel-cookies"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.bbcgoodfood.com/recipes/9088/bestever-chunky-guacamole"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.bbcgoodfood.com/recipes/10603/roasted-vegetable-lasagne"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.bbcgoodfood.com/recipes/358608/cauliflower-cheese"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.bbcgoodfood.com/recipes/1267639/peppered-mackerel-new-potato-and-watercress-salad"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.bbcgoodfood.com/recipes/1396633/courgette-broccoli-and-gremolata-pasta"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.bbcgoodfood.com/recipes/2220641/squid-samphire-and-pea-salad"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.bbcgoodfood.com/recipes/2358637/sausage-fennel-and-rocket-fusilli"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.bbcgoodfood.com/recipes/2493639/spanish-stuffed-marrow"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.bbcgoodfood.com/recipes/asparagus-chilli-feta-farfalle"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.bbcgoodfood.com/recipes/broad-bean-feta-watercress-salad"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.bbcgoodfood.com/recipes/burrata-bruschetta-summer-beans"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.bbcgoodfood.com/recipes/butter-poached-asparagus-leeks-peas"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.bbcgoodfood.com/recipes/eton-mess-cheesecake"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.bbcgoodfood.com/recipes/frozen-blackberry-yogurt"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.bbcgoodfood.com/recipes/gnocchi-parsley-butter-samphire"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.bbcgoodfood.com/recipes/italian-sausage-chestnut-pasta"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.bbcgoodfood.com/recipes/new-potatoes-spinach-capers"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.bbcgoodfood.com/recipes/passionfruit-pornstar-martini"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.bbcgoodfood.com/recipes/pea-broad-bean-shakshuka"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.bbcgoodfood.com/recipes/tomato-burrata-broad-bean-salad"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.bbcgoodfood.com/recipes/ultimate-tomato-salsa"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.bonappetit.com/recipe/grilled-artichokes-with-yogurt-dill-dipping-sauce"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.bonappetit.com/recipe/turnip-and-kale-gratin"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.bosh.tv/recipes/butternut-squash-dahl-veganuary-week-2"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.bosh.tv/recipes/quick-and-easy-moroccan-couscous"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.channel4.com/programmes/sunday-brunch/articles/all/cauliflower-katsu/6407"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.chilipeppermadness.com/chili-pepper-recipes/breakfasts/shakshuka-eggs-poached-in-spicy-tomato-sauce/"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.chinasichuanfood.com/chinese-style-cabbage-stir-fry/"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.connoisseurusveg.com/gingery-carrot-stew-with-peanuts-and-cilantro/"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.delicious.com.au/recipes/mango-avocado-black-lentil-salad/HsSi2VPv?r=recipes/collections/mangorecipes"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.deliciousmagazine.co.uk/recipes/roast-potatoes-with-sage/"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.delish.com/holiday-recipes/thanksgiving/a23104970/apple-pie-bars-recipe/"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.dinneratthezoo.com/ground-beef-tacos/"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.donnahay.com.au/recipes/banana-bread"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.finecooking.com/recipe/broccoflower-and-cremini-mushrooms-with-garlic-and-rosemary"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.foodandwine.com/recipes/broccoli-frittata"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.foodandwine.com/recipes/spicy-carrot-sandwiches"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.foodiequine.co.uk/2015/07/strawberry-blackcurrant-coconut-water.html"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.foodnetwork.ca/recipe/grilled-peach-panzanella-salad/18397/"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.greatbritishchefs.com/recipes/chicory-recipe"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.healthyseasonalrecipes.com/simple-kohlrabi-with-pistachios-and-sage/"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.healthyseasonalrecipes.com/simple-sauteed-swiss-chard/"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.jamieoliver.com/recipes/potato-recipes/easy-rustic-gnocchi/"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.jamieoliver.com/recipes/potato-recipes/easy-rustic-gnocchi/"} recipe appears to still exist 2`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.jamieoliver.com/recipes/vegetables-recipes/ratatouille/"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.kitchenhealssoul.com/red-currant-muffins/"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.marthastewart.com/1047287/buckwheat-waffles-fresh-ricotta-and-citrus"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.mommypotamus.com/elderberry-jam-recipe/"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.momskitchenhandbook.com/uncategorized/pasta-with-creamy-beet-sauce-walnuts-and-parmesan/"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.mydiaryofus.com/blog/2015/5/14/plum-and-thyme-prosecco-smash"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.myrecipes.com/recipe/escarole-with-bacon-white-beans"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.myrecipes.com/recipe/green-beans-with-goat-cheese-tomatoes-almonds"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.olivemagazine.com/recipes/family/cherry-jam-and-almond-galettes/"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.olivemagazine.com/recipes/quick-and-easy/caramel-poached-apricots/"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.pillsbury.com/recipes/cucumber-pepper-salad/1159ed6b-7863-4b60-b4e1-ce95520c80dc"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.recipetineats.com/rocket-aragula-beetroot-walnuts-feta-wbalsamic-dressing/"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.riverford.co.uk/recipes/view/recipe/runner-bean-ratatouille"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.sbs.com.au/food/recipes/blackcurrant-jam"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.sbs.com.au/food/recipes/spicy-korean-noodles-cucumber-and-egg-bibim-guksu"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.sbs.com.au/food/recipes/swede-and-silverbeet-pie"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.taste.com.au/recipes/fig-ginger-gratin/inu15was"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.taste.com.au/recipes/ratatouille-5/44db8695-c005-4db1-b874-ba3ffa7c3116"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.tasteofhome.com/recipes/tomato-garlic-butter-bean-dinner/"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.tasteofhome.com/recipes/warm-rice-pintos-salad/"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.tastymediterraneo.com/tagliatelle-with-fresh-oregano-pesto/"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.thekitchn.com/how-to-make-stuffed-peppers-234387"} recipe appears to still exist 1`] = `403`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.theocooks.com/spicy-aubergine-stir-fry/"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.thespruceeats.com/asian-pear-crisp-2216135"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.thespruceeats.com/kale-and-cabbage-slaw-with-dijon-mustard-vinaigrette-4123786"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.thespruceeats.com/sauteed-radicchio-recipe-2217562"} recipe appears to still exist 1`] = `200`;

exports[`check recipes sites have not changed {"linkUrl": "https://www.waitrose.com/home/recipes/recipe_directory/s/scorched-leeks-withtomatoandcaperdressing.html"} recipe appears to still exist 1`] = `200`;
